File: test7Go/oop/models/pizza/simplePizza.go
package pizza

import (
	oop "oop/models"
)

type SimplePizza struct {
	Topping string
	oop.CommandItem
}

func NewSimplePizza(name string, price float64) SimplePizza {
	topping := "cheese, sauce"
	commandItem := oop.CommandItem{Name: name, Price: price, Quantity: 1}

	return SimplePizza{topping, commandItem}
}

File: test7C#/test7/Model/Pizza/SimplePizza.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace test7.Model
{
    class SimplePizza: CommandItem
    {
        private string topping;

        public SimplePizza(string name, Decimal price): base(name, price, 1)
        {
            this.topping = "cheese, sauce";
        }

        public string Topping()
        {
            return topping;
        }

        public void Topping(string topping)
        {
            this.topping = topping;
        }
    }
}
